apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -7,6 +7,16 @@ import { ViewType } from '@cdo/apps/code-studio/stageLockRedux';
 import AssignToSection from '@cdo/apps/templates/courseOverview/AssignToSection';
 import experiments, { SECTION_FLOW_2017 } from '@cdo/apps/util/experiments';
 
+export const NOT_STARTED = 'NOT_STARTED';
+export const IN_PROGRESS = 'IN_PROGRESS';
+export const COMPLETED = 'COMPLETED';
+
+const NEXT_BUTTON_TEXT = {
+  NOT_STARTED: i18n.tryNow(),
+  IN_PROGRESS: i18n.continue(),
+  COMPLETED: i18n.printCertificate(),
```

Comment:
nit; these keys are referring to the strings and not the consts; I don't think that's a huge deal, but it makes me slightly uncomfortable; wanna update these to `[NOT_STARTED]: `-style declarations?